docs/architecture/adr-104-out-of-band-state-sync.md

```diff
@@ -124,58 +138,173 @@ in order to adopt this solution.
 
 ## Decision
```

Comment:
I would maintain the previous content of this section, which provides a high-level overview of the proposed solution, to then include the observations you added.